server/app/services/cloud/azure/AzurePublicStorage.java

```diff
@@ -1,25 +1,70 @@
 package services.cloud.azure;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableSet;
+import com.typesafe.config.Config;
+import java.util.Optional;
+import java.util.UUID;
+import javax.inject.Inject;
 import services.cloud.PublicStorageClient;
 import services.cloud.StorageUploadRequest;
 
-/** An Azure Blob Storage implementation of public storage. */
+/**
+ * An Azure Blob Storage implementation of public storage.
+ *
+ * <p>TODO(#9213): Complete program card image upload for azure, as is, this is a no-op, test-only
+ * implementation of the AzurePublicStorage class.
+ */
 public class AzurePublicStorage extends PublicStorageClient {
+  public static final String AZURE_STORAGE_ACCT_CONF_PATH = "azure.blob.account";
+
+  @VisibleForTesting
+  static final String AZURE_PUBLIC_CONTAINER_CONF_PATH = "azure.blob.public_container";
+
+  private final String container;
```

Comment:
```suggestion
  static final String AZURE_PUBLIC_CONTAINER_CONF_PATH = "azure.blob.public_container_name";

  private final String containerName;
```

Reply:
just checking about this -- if we go with public_contain_name .. naming will vary from the existing azure naming convention.. where the private applicant storage vars are named

azure.blob.container

and the env var is 

AZURE_STORAGE_ACCOUNT_CONTAINER

Reply:
done